docs/reference/architecture/hosts_vms.md

```diff
@@ -13,58 +13,49 @@ products:
-
-## Deployment scenarios
-
-See the recommended architectures per Elastic deployment scenarios:
+2.  Enriching application telemetry from OTel SDKs with the corresponding hosts resource attributes.
 
 :::{note}
-Elastic's Observability solution is technically compatible with edge setups that are fully based on contrib OTel components as long as the ingestion path follows the recommendations outlined in the following sections.
+Elastic Observability is technically compatible with edge setups that are fully based on contrib OTel components as long as the ingestion path follows the recommendations outlined in the following sections.
 :::
 
-### Elastic Cloud Serverless
-
-{{serverless-full}} provides a [Managed OTLP Endpoint](/reference/motlp.md) for ingestion of OpenTelemetry data.
-
-![VM-Serverless](../images/arch-vm-serverless.png)
+## Deployment scenarios
 
-Users can send their OTel data from the edge setup in OTel-native format through OTLP without any additional requirements for self-managed preprocessing of data.
+The following sections outline the recommended architectures for different Elastic deployment scenarios.
 
-### Elastic Cloud Hosted
+### {{serverless-full}}
 
-As of Elastic Stack version 9.0 on {{ech}} (ECH), you need to run a self-hosted EDOT Collector in Gateway Mode to ingest OTel data from the edge setup in OTel-native format into the Elastic-hosted {{es}}.
+{{serverless-full}} provides a [Managed OTLP Endpoint](/reference/motlp.md) for ingestion of OpenTelemetry data.
 
-![VM-ECH](../images/arch-vm-ech.png)
+![VM-Serverless](../images/host-serverless.png)
 
-The EDOT Collector in Gateway mode enriches and pre-aggregates the data for a seamless experience in the Elastic Observability solution before ingesting it directly into {{es}}.
+Users can send data direct from the Collectors or SDKs deployed on the edge environment through OTLP without any additional requirements for managing an ingestion layer.
 
-If required, users can build their custom, EDOT-like Collector
-[following these instructions](elastic-agent://reference/edot-collector/custom-collector.md).
+### {{ech}}
+```{applies_to}
+ess:
+stack: preview 9.2
+```
 
-:::{note}
-The EDOT Gateway Collector does not send data through Elastic's Integration / APM Server on ECH to ingest data into {{es}}.
-:::
+{{ech}} provides a [Managed OTLP Endpoint](/reference/motlp.md) for ingestion of OpenTelemetry data. Users can send data direct from the Collectors or SDKs deployed on the edge environment through OTLP without any additional requirements for managing an ingestion layer.
 
-:::{important}
-If self-managing an EDOT Gateway is not a valid option for you, refer to [Elastic's classic ingestion path for OTel data on ECH](docs-content://solutions/observability/apm/use-opentelemetry-with-apm.md).
-:::
+![VM-ech](../images/host-ech.png)
```

Comment:
The text and image for this section are not aligned.

Text indicates that users should use the managed OTLP endpoint for ECH, but the image still shows the Gateway as a required component (though with the annotation that it won't be required once mOTLP is available). But the text indicates that mOTLP is already available on ECH.

It's for @mlunadia to decide how what we want to actually recommend here, but maybe we should also call out the mOTLP on ECH is in tech preview.